zebedee-cms/src/main/java/com/github/onsdigital/zebedee/api/Collection.java

```diff
@@ -130,10 +132,28 @@ public CollectionDescription get(HttpServletRequest request, HttpServletResponse
         return result;
     }
 
+    private void requireViewPermission(Session session, String collectionId) throws UnauthorizedException {
+        boolean canView = false;
+        try {
+            canView = permissionsService.canView(session, collectionId);
+        } catch (IOException ex) {
+            throw new UnauthorizedException("You are not authorised to view this collection");
+        }
+
+        if (!canView) {
+            warn().user(session.getEmail())
+                    .collectionID(collectionId)
+                    .log("request unsuccessful user denied view permission");
```

Comment:
I think we need to be smarter around what we log in Zebedee as its quite noisy. At present this will log the warning and eventually the exception stack trace. We don't need both so could we remove the warning log and update the exception below to include the user email and collection details. Something like:

```
throw new UnauthorizedException(MessageFormat.format("User is {0} not authorised to view collection {1}", session.getEmail, collectionID)
```
We get all the details we required but only once.